archive/app/controllers/ArchiveController.scala

```diff
@@ -144,7 +144,7 @@ object ArchiveController extends Controller with Logging with ExecutionContexts
     request.headers.get("User-Agent").getOrElse("no user agent") match {
       case GoogleBot(_) =>
         log.warn(s"404,${RequestLog(request)}")
-        Googlebot404Count.increment()
+        app.Global.Googlebot404Count.increment()
```

Comment:
Rather than accessing `Googlebot404Count` via `Global`, you could move it into a companion object `ArchiveMetrics`.